packages/arb-token-bridge-ui/src/token-bridge-sdk/utils.ts

```diff
@@ -28,25 +29,23 @@ export const getBridgeTransferProperties = (
   const sourceChainId = props.sourceChainId
   const destinationChainId = props.destinationChainId
 
-  const isSourceChainEthereumMainnetOrTestnet =
-    isNetwork(sourceChainId).isEthereumMainnetOrTestnet
   const isDestinationChainEthereumMainnetOrTestnet =
     isNetwork(destinationChainId).isEthereumMainnetOrTestnet
 
   const isSourceChainArbitrum = isNetwork(sourceChainId).isArbitrum
   const isDestinationChainArbitrum = isNetwork(destinationChainId).isArbitrum
 
   const isSourceChainOrbit = isNetwork(sourceChainId).isOrbitChain
-  const isDestinationChainOrbit = isNetwork(destinationChainId).isOrbitChain
 
-  const isDeposit =
-    isSourceChainEthereumMainnetOrTestnet ||
-    (isSourceChainArbitrum && isDestinationChainOrbit)
+  const { isBase: isDestinationChainBase } = isNetwork(destinationChainId)
+
+  const isDeposit = isDepositMode({ sourceChainId, destinationChainId })
 
   const isWithdrawal =
```

Comment:
Wouldn't this work?
```typescript
const isWithdrawal = !isDeposit
```

Reply:
checking with @dewanshparashar

Reply:
No we cannot. Because our `isSupported` flag in bridge-sdk>utils is a union of `isDeposit || isWithdrawal || isTeleport`  and having `isWthdrawal = !isDeposit` will lead it to support all unsupported combinations.